src/plugins/ButtonSizePlugin.js

```diff
@@ -0,0 +1,108 @@
+import { BasePlugin } from './BasePlugin';
+import { Slider } from '../ui-elements/Slider';
+
+/**
+ * @export
+ * @class ButtonSizePlugin
+ * @extends {BasePlugin}
+ *
+ */
+export class ButtonSizePlugin extends BasePlugin {
+  /**
+   *Creates an instance of ButtonSizePlugin.
+   * @param {object} params
+   * @param {string | HTMLElement} [params.buttonSliders]
+   * @param {number} [params.defaultButtonSize=0.5]
+   * @memberof ButtonSizePlugin
+   */
+  constructor({
```

Comment:
I think we discussed this, but it might not be immediately clear in a ticket: Can we change the constructor here?

```javascript
new ButtonSizePlugin('input.button-size[type=range]', { defaultValue: 0.5 });
```